src/Twilio/Clients/NoAuth/TwilioNoAuthRestClient.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System;
+using System.Net;
+using System.Linq;
+using Newtonsoft.Json;
+using Twilio.Exceptions;
+using Twilio.Http.BearerToken;
+using Twilio.Jwt;
+
+
+#if !NET35
+using System.Threading.Tasks;
+#endif
+
+using Twilio.Http;
+using Twilio.Http.NoAuth;
+#if NET35
+using Twilio.Http.Net35;
+#endif
+
+
+namespace Twilio.Clients.NoAuth
+{
+    /// <summary>
+    /// Implementation of a TwilioRestClient.
+    /// </summary>
+    public class TwilioNoAuthRestClient
+    {
+        /// <summary>
+        /// Client to make HTTP requests
+        /// </summary>
+        public NoAuthHttpClient HttpClient { get; }
+
+        /// <summary>
+        /// Twilio region to make requests to
+        /// </summary>
+        public string Region { get; }
+
+        /// <summary>
+        /// Twilio edge to make requests to
+        /// </summary>
+        public string Edge { get; set; }
+
+        /// <summary>
+        /// Additions to the user agent string
+        /// </summary>
+        public string[] UserAgentExtensions { get; set; }
+
+        /// <summary>
+        /// Log level for logging
+        /// </summary>
+        public string LogLevel { get; set; } = Environment.GetEnvironmentVariable("TWILIO_LOG_LEVEL");
+
+        /// <summary>
+        /// Constructor for a TwilioRestClient
+        /// </summary>
+        ///
+        /// <param name="accessToken">access token for requests</param>
+        /// <param name="accountSid">account sid to make requests for</param>
+        /// <param name="region">region to make requests for</param>
```

Comment:
Do we need to accessToken and accountSid here ?

Reply:
No, removed